test/helpers/test/time.test.js

```diff
@@ -0,0 +1,65 @@
+const time = require('../time');
+const shouldFail = require('../shouldFail');
+
+const BigNumber = web3.BigNumber;
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+describe('time', function () {
+  beforeEach(async function () {
+    this.start = await time.latest();
+  });
+
+  describe('increase', function () {
+    it('increases time by a duration', async function () {
+      await time.increase(time.duration.hours(1));
+
+      const end = this.start + time.duration.hours(1);
+      (await time.latest()).should.be.within(end - 20, end + 20); // +-20 sec tolerance
+    });
+
+    it('throws with negative durations', async function () {
+      await shouldFail(time.increase(-1));
+    });
+  });
+
+  describe('increaseTo', function () {
+    it('increases time to a time in the future', async function () {
+      await time.increaseTo(this.start + time.duration.hours(1));
```

Comment:
Why not use the `end` variable defined below?

Reply:
Because it seems I'm not very smart :sweat_smile: